.eslintrc.js

```diff
@@ -13,4 +13,5 @@ module.exports = {
       },
     },
   ],
+  ignorePatterns: ['tests/fixtures/**/*'],
```

Comment:
Instead of adding an `ignorePatterns` rule, could we not remove `tests` from [here](https://github.com/netlify/zip-it-and-ship-it/blob/main/package.json#L28)?

Reply:
I think we should keep actual test files linted, as tests are code and should be maintainable. The fixtures are more resources than code, which is why I think we shouldn't be linting them.

Curious, what's the reason we have eslint config in `package.json`? Instead of adding the ignore pattern there, we could also think about moving that config into `.eslintrc.js`.

Reply:
> I think we should keep actual test files linted, as tests are code and should be maintainable. The fixtures are more resources than code, which is why I think we shouldn't be linting them.

Doh! That makes sense, ignore my comment.

> 
> Curious, what's the reason we have eslint config in `package.json`? Instead of adding the ignore pattern there, we could also think about moving that config into `.eslintrc.js`.

I'm not sure, to be honest. That's the pattern we use in our [Node project template](https://github.com/netlify/node-template/blob/main/%7B%7Bcookiecutter.project_slug%7D%7D/package.json#L26-L29). I'm not against changing it, but perhaps we should discuss that with the guild so that we use a unified approach.